api/Navigator.json

```diff
@@ -2201,7 +2202,7 @@
               "version_added": "12"
             },
             "firefox": {
-              "version_added": true
+              "version_added": "1"
             },
             "firefox_android": {
               "version_added": true
```

Comment:
I see firefox_android wasn't updated here. That's OK, we'll get to it later, but might point to a problem with some script or method.